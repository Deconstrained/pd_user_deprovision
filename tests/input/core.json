{
  "get_user_id": [
    "lucas@pagerduty.com",
    "abel.tuter@example.com"
  ],
  "list_users_on_team": [
    "PU30AUS",
    "PRNBCAY"
  ],
  "list_user_escalation_policies": [
    "PMPOKBU"
  ],
  "get_schedule": [
    "PVUNMZG"
  ],
  "get_escalation_policy": [
    "PUT4PSP"
  ],
  "check_schedule_for_user": [
    {
      "user_id": "ABCDEF",
      "schedule": {
        "users": [
          {
            "id": "AAAAA",
            "type": "user",
            "name": "Arya Stark"
          },
          {
            "id": "BBBBBB",
            "type": "user",
            "name": "Sansa Stark"
          },
          {
            "id": "ABCDEF",
            "type": "user",
            "name": "Jon Snow"
          },
          {
            "id": "CCCCCC",
            "type": "user",
            "name": "Robb Stark"
          }
        ]
      }
    },
    {
      "user_id": "ABCDEF",
      "schedule": {
        "users": [
          {
            "id": "AAAAA",
            "type": "user",
            "name": "Arya Stark"
          },
          {
            "id": "BBBBBB",
            "type": "user",
            "name": "Sansa Stark"
          },
          {
            "id": "CCCCCC",
            "type": "user",
            "name": "Jon Snow"
          },
          {
            "id": "DDDDDD",
            "type": "user",
            "name": "Robb Stark"
          }
        ]
      }
    }
  ],
  "check_team_for_user": [
    {
      "user_id": "ABCDEF",
      "team_users": [
        {
          "id": "AAAAA",
          "type": "user",
          "name": "Arya Stark"
        },
        {
          "id": "BBBBBB",
          "type": "user",
          "name": "Sansa Stark"
        },
        {
          "id": "ABCDEF",
          "type": "user",
          "name": "Jon Snow"
        },
        {
          "id": "CCCCCC",
          "type": "user",
          "name": "Robb Stark"
        }
      ]
    },
    {
      "user_id": "ABCDEF",
      "team_users": [
        {
          "id": "AAAAA",
          "type": "user",
          "name": "Arya Stark"
        },
        {
          "id": "BBBBBB",
          "type": "user",
          "name": "Sansa Stark"
        },
        {
          "id": "CCCCCC",
          "type": "user",
          "name": "Jon Snow"
        },
        {
          "id": "DDDDDD",
          "type": "user",
          "name": "Robb Stark"
        }
      ]
    }
  ],
  "get_user_layer_index": [
    {
      "user_id": "ABCDEF",
      "schedule_layer": {
        "id": "XXXXXX",
        "name": "Layer 1",
        "users": [
          {
            "user": {
              "id": "AAAAAA",
              "type": "user",
              "name": "Arya Stark"
            }
          },
          {
            "user": {
              "id": "ABCDEF",
              "type": "user",
              "name": "Jon Snow"
            }
          },
          {
            "user": {
              "id": "BBBBBB",
              "type": "user",
              "name": "Sansa Stark"
            }
          }
        ]
      }
    },
    {
      "user_id": "ABCDEF",
      "schedule_layer": {
        "id": "XXXXXX",
        "name": "Layer 1",
        "users": [
          {
            "user": {
              "id": "AAAAAA",
              "type": "user",
              "name": "Arya Stark"
            }
          },
          {
            "user": {
              "id": "BBBBBB",
              "type": "user",
              "name": "Sansa Stark"
            }
          },
          {
            "user": {
              "id": "CCCCCC",
              "type": "user",
              "name": "Robb Stark"
            }
          }
        ]
      }
    }
  ],
  "get_target_indices": [
    {
      "id": "ABCDEF",
      "escalation_rules": [
        {
          "id": "XXXXXX",
          "targets": [
            {
              "id": "AAAAAA"
            },
            {
              "id": "ABCDEF"
            },
            {
              "id": "BBBBBB"
            }
          ]
        },
        {
          "id": "YYYYYY",
          "targets": [
            {
              "id": "CCCCCC"
            },
            {
              "id": "DDDDDD"
            }
          ]
        },
        {
          "id": "ZZZZZZ",
          "targets": [
            {
              "id": "ABCDEF"
            }
          ]
        }
      ]
    },
    {
      "id": "ABCDEF",
      "escalation_rules": [
        {
          "id": "XXXXXX",
          "targets": [
            {
              "id": "AAAAAA"
            },
            {
              "id": "BBBBBB"
            },
            {
              "id": "CCCCCC"
            },
            {
              "id": "DDDDDD"
            }
          ]
        },
        {
          "id": "YYYYYY",
          "targets": [
            {
              "id": "EEEEEE"
            },
            {
              "id": "FFFFFF"
            },
            {
              "id": "ABCDEF"
            },
            {
              "id": "GGGGGG"
            }
          ]
        }
      ]
    }
  ]
}
