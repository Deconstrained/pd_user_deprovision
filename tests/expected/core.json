{
  "get_user_id": [
    "P9GJP78",
    "P05F6B9"
  ],
  "list_users_on_team": [
    [
      {
        "name": "Delete User Test User",
        "email": "lucas+deletetestuser@pagerduty.com",
        "time_zone": "America/Los_Angeles",
        "color": "royal-blue",
        "avatar_url": "https://secure.gravatar.com/avatar/da2941dfe199a49605722f3851921526.png?d=mm&r=PG",
        "billed": true,
        "role": "user",
        "description": null,
        "invitation_sent": true,
        "contact_methods": [
          {
            "id": "PFBAAXZ",
            "type": "email_contact_method_reference",
            "summary": "Default",
            "self": "https://api.pagerduty.com/users/PMPOKBU/contact_methods/PFBAAXZ",
            "html_url": null
          }
        ],
        "notification_rules": [
          {
            "id": "PDPBSHZ",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PFBAAXZ",
            "self": "https://api.pagerduty.com/users/PMPOKBU/notification_rules/PDPBSHZ",
            "html_url": null
          },
          {
            "id": "P1C1TM5",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PFBAAXZ",
            "self": "https://api.pagerduty.com/users/PMPOKBU/notification_rules/P1C1TM5",
            "html_url": null
          }
        ],
        "job_title": null,
        "teams": [
          {
            "id": "PU30AUS",
            "type": "team_reference",
            "summary": "Delete User Test Team",
            "self": "https://api.pagerduty.com/teams/PU30AUS",
            "html_url": "https://pdt-lucas.pagerduty.com/teams/PU30AUS"
          }
        ],
        "coordinated_incidents": [],
        "id": "PMPOKBU",
        "type": "user",
        "summary": "Delete User Test User",
        "self": "https://api.pagerduty.com/users/PMPOKBU",
        "html_url": "https://pdt-lucas.pagerduty.com/users/PMPOKBU"
      }
    ],
    [
      {
        "name": "Aeron Greyjoy",
        "email": "lucas+thethricedrowned@pagerduty.com",
        "time_zone": "America/Los_Angeles",
        "color": "lime-green",
        "avatar_url": "https://secure.gravatar.com/avatar/64d3a16c5ae9b6c7ccb263f3024ecf4d.png?d=mm&r=PG",
        "billed": true,
        "role": "user",
        "description": null,
        "invitation_sent": true,
        "contact_methods": [
          {
            "id": "PARPSEN",
            "type": "email_contact_method_reference",
            "summary": "Default",
            "self": "https://api.pagerduty.com/users/PMOLFJG/contact_methods/PARPSEN",
            "html_url": null
          },
          {
            "id": "P1BVOW4",
            "type": "phone_contact_method_reference",
            "summary": "Mobile",
            "self": "https://api.pagerduty.com/users/PMOLFJG/contact_methods/P1BVOW4",
            "html_url": null
          },
          {
            "id": "PPWSQA7",
            "type": "sms_contact_method_reference",
            "summary": "Mobile",
            "self": "https://api.pagerduty.com/users/PMOLFJG/contact_methods/PPWSQA7",
            "html_url": null
          }
        ],
        "notification_rules": [
          {
            "id": "PC4M0WH",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PARPSEN",
            "self": "https://api.pagerduty.com/users/PMOLFJG/notification_rules/PC4M0WH",
            "html_url": null
          },
          {
            "id": "P0UP1BQ",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PARPSEN",
            "self": "https://api.pagerduty.com/users/PMOLFJG/notification_rules/P0UP1BQ",
            "html_url": null
          },
          {
            "id": "P3S4JOB",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel P1BVOW4",
            "self": "https://api.pagerduty.com/users/PMOLFJG/notification_rules/P3S4JOB",
            "html_url": null
          },
          {
            "id": "PF2EI75",
            "type": "assignment_notification_rule_reference",
            "summary": "3 minutes: channel P1BVOW4",
            "self": "https://api.pagerduty.com/users/PMOLFJG/notification_rules/PF2EI75",
            "html_url": null
          },
          {
            "id": "PRN1LSE",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PPWSQA7",
            "self": "https://api.pagerduty.com/users/PMOLFJG/notification_rules/PRN1LSE",
            "html_url": null
          }
        ],
        "job_title": null,
        "teams": [
          {
            "id": "P9XAZQQ",
            "type": "team_reference",
            "summary": "The first team",
            "self": "https://api.pagerduty.com/teams/P9XAZQQ",
            "html_url": "https://pdt-lucas.pagerduty.com/teams/P9XAZQQ"
          },
          {
            "id": "PRNBCAY",
            "type": "team_reference",
            "summary": "Delete User Test Team 2",
            "self": "https://api.pagerduty.com/teams/PRNBCAY",
            "html_url": "https://pdt-lucas.pagerduty.com/teams/PRNBCAY"
          }
        ],
        "coordinated_incidents": [],
        "id": "PMOLFJG",
        "type": "user",
        "summary": "Aeron Greyjoy",
        "self": "https://api.pagerduty.com/users/PMOLFJG",
        "html_url": "https://pdt-lucas.pagerduty.com/users/PMOLFJG"
      },
      {
        "name": "Alejandra Prenatt",
        "email": "alejandra.prenatt@example.com",
        "time_zone": "America/Los_Angeles",
        "color": "dark-cyan",
        "avatar_url": "https://secure.gravatar.com/avatar/69615a38d005aabd9f1c73fee28ecd7f.png?d=mm&r=PG",
        "billed": true,
        "role": "limited_user",
        "description": null,
        "invitation_sent": true,
        "contact_methods": [
          {
            "id": "PWXVQ23",
            "type": "email_contact_method_reference",
            "summary": "Default",
            "self": "https://api.pagerduty.com/users/P0HN8MG/contact_methods/PWXVQ23",
            "html_url": null
          }
        ],
        "notification_rules": [
          {
            "id": "PODTFFF",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PWXVQ23",
            "self": "https://api.pagerduty.com/users/P0HN8MG/notification_rules/PODTFFF",
            "html_url": null
          },
          {
            "id": "PZYQHJI",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PWXVQ23",
            "self": "https://api.pagerduty.com/users/P0HN8MG/notification_rules/PZYQHJI",
            "html_url": null
          }
        ],
        "job_title": null,
        "teams": [
          {
            "id": "PRNBCAY",
            "type": "team_reference",
            "summary": "Delete User Test Team 2",
            "self": "https://api.pagerduty.com/teams/PRNBCAY",
            "html_url": "https://pdt-lucas.pagerduty.com/teams/PRNBCAY"
          }
        ],
        "coordinated_incidents": [],
        "id": "P0HN8MG",
        "type": "user",
        "summary": "Alejandra Prenatt",
        "self": "https://api.pagerduty.com/users/P0HN8MG",
        "html_url": "https://pdt-lucas.pagerduty.com/users/P0HN8MG"
      },
      {
        "name": "Luke Epp",
        "email": "lucas@pagerduty.com",
        "time_zone": "America/Los_Angeles",
        "color": "blue",
        "avatar_url": "https://secure.gravatar.com/avatar/bf80d8d124f3c9fba51b5b65f0ce0388.png?d=mm&r=PG",
        "billed": true,
        "role": "owner",
        "description": null,
        "invitation_sent": false,
        "contact_methods": [
          {
            "id": "PLWSUYQ",
            "type": "email_contact_method_reference",
            "summary": "Default",
            "self": "https://api.pagerduty.com/users/P9GJP78/contact_methods/PLWSUYQ",
            "html_url": null
          },
          {
            "id": "PYNRCPN",
            "type": "email_contact_method_reference",
            "summary": "Work",
            "self": "https://api.pagerduty.com/users/P9GJP78/contact_methods/PYNRCPN",
            "html_url": null
          },
          {
            "id": "PNSUA87",
            "type": "email_contact_method_reference",
            "summary": "Work",
            "self": "https://api.pagerduty.com/users/P9GJP78/contact_methods/PNSUA87",
            "html_url": null
          },
          {
            "id": "PZXZTLJ",
            "type": "phone_contact_method_reference",
            "summary": "Mobile",
            "self": "https://api.pagerduty.com/users/P9GJP78/contact_methods/PZXZTLJ",
            "html_url": null
          },
          {
            "id": "P426GHP",
            "type": "push_notification_contact_method_reference",
            "summary": "iPhone (3)",
            "self": "https://api.pagerduty.com/users/P9GJP78/contact_methods/P426GHP",
            "html_url": null
          },
          {
            "id": "POCCRHG",
            "type": "sms_contact_method_reference",
            "summary": "Mobile",
            "self": "https://api.pagerduty.com/users/P9GJP78/contact_methods/POCCRHG",
            "html_url": null
          }
        ],
        "notification_rules": [
          {
            "id": "PGBCFD1",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PLWSUYQ",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/PGBCFD1",
            "html_url": null
          },
          {
            "id": "P512GHX",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PLWSUYQ",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/P512GHX",
            "html_url": null
          },
          {
            "id": "PK43JWI",
            "type": "assignment_notification_rule_reference",
            "summary": "1 minutes: channel PLWSUYQ",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/PK43JWI",
            "html_url": null
          },
          {
            "id": "PD0SEYW",
            "type": "assignment_notification_rule_reference",
            "summary": "18 minutes: channel PLWSUYQ",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/PD0SEYW",
            "html_url": null
          },
          {
            "id": "P3AFN90",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PYNRCPN",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/P3AFN90",
            "html_url": null
          },
          {
            "id": "PFOS94U",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel PNSUA87",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/PFOS94U",
            "html_url": null
          },
          {
            "id": "POR9FOC",
            "type": "assignment_notification_rule_reference",
            "summary": "2 minutes: channel PZXZTLJ",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/POR9FOC",
            "html_url": null
          },
          {
            "id": "PVOZ4UK",
            "type": "assignment_notification_rule_reference",
            "summary": "5 minutes: channel PZXZTLJ",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/PVOZ4UK",
            "html_url": null
          },
          {
            "id": "PTPRZS2",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel P426GHP",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/PTPRZS2",
            "html_url": null
          },
          {
            "id": "P1Y3Q38",
            "type": "assignment_notification_rule_reference",
            "summary": "2 minutes: channel P426GHP",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/P1Y3Q38",
            "html_url": null
          },
          {
            "id": "PG6W6Z0",
            "type": "assignment_notification_rule_reference",
            "summary": "4 minutes: channel P426GHP",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/PG6W6Z0",
            "html_url": null
          },
          {
            "id": "P8TTKBR",
            "type": "assignment_notification_rule_reference",
            "summary": "0 minutes: channel POCCRHG",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/P8TTKBR",
            "html_url": null
          },
          {
            "id": "PZ96HSF",
            "type": "assignment_notification_rule_reference",
            "summary": "1 minutes: channel POCCRHG",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/PZ96HSF",
            "html_url": null
          },
          {
            "id": "PSQG9HZ",
            "type": "assignment_notification_rule_reference",
            "summary": "3 minutes: channel POCCRHG",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/PSQG9HZ",
            "html_url": null
          },
          {
            "id": "P4VJ7DW",
            "type": "assignment_notification_rule_reference",
            "summary": "5 minutes: channel POCCRHG",
            "self": "https://api.pagerduty.com/users/P9GJP78/notification_rules/P4VJ7DW",
            "html_url": null
          }
        ],
        "job_title": null,
        "teams": [
          {
            "id": "PD28XX6",
            "type": "team_reference",
            "summary": "The best team",
            "self": "https://api.pagerduty.com/teams/PD28XX6",
            "html_url": "https://pdt-lucas.pagerduty.com/teams/PD28XX6"
          },
          {
            "id": "PWQ8081",
            "type": "team_reference",
            "summary": "Engineering",
            "self": "https://api.pagerduty.com/teams/PWQ8081",
            "html_url": "https://pdt-lucas.pagerduty.com/teams/PWQ8081"
          },
          {
            "id": "PRNBCAY",
            "type": "team_reference",
            "summary": "Delete User Test Team 2",
            "self": "https://api.pagerduty.com/teams/PRNBCAY",
            "html_url": "https://pdt-lucas.pagerduty.com/teams/PRNBCAY"
          }
        ],
        "coordinated_incidents": [],
        "id": "P9GJP78",
        "type": "user",
        "summary": "Luke Epp",
        "self": "https://api.pagerduty.com/users/P9GJP78",
        "html_url": "https://pdt-lucas.pagerduty.com/users/P9GJP78"
      }
    ]
  ],
  "list_user_escalation_policies": [
    [
      {
        "id": "PUT4PSP",
        "type": "escalation_policy",
        "summary": "Delete User Test EP",
        "self": "https://api.pagerduty.com/escalation_policies/PUT4PSP",
        "html_url": "https://pdt-lucas.pagerduty.com/escalation_policies/PUT4PSP",
        "name": "Delete User Test EP",
        "escalation_rules": [
          {
            "id": "P2MLJ1X",
            "escalation_delay_in_minutes": 30,
            "targets": [
              {
                "id": "PMPOKBU",
                "type": "user_reference",
                "summary": "Delete User Test User",
                "self": "https://api.pagerduty.com/users/PMPOKBU",
                "html_url": "https://pdt-lucas.pagerduty.com/users/PMPOKBU"
              }
            ]
          }
        ],
        "services": [],
        "num_loops": 0,
        "teams": [
          {
            "id": "PU30AUS",
            "type": "team_reference",
            "summary": "Delete User Test Team",
            "self": "https://api.pagerduty.com/teams/PU30AUS",
            "html_url": "https://pdt-lucas.pagerduty.com/teams/PU30AUS"
          }
        ],
        "description": "",
        "privilege": null
      }
    ]
  ],
  "get_schedule": [
    {
      "id": "PVUNMZG",
      "type": "schedule",
      "summary": "Delete User Test Schedule",
      "self": "https://api.pagerduty.com/schedules/PVUNMZG",
      "html_url": "https://pdt-lucas.pagerduty.com/schedules/PVUNMZG",
      "name": "Delete User Test Schedule",
      "time_zone": "America/Los_Angeles",
      "description": null,
      "escalation_policies": [],
      "users": [
        {
          "id": "PMPOKBU",
          "type": "user_reference",
          "summary": "Delete User Test User",
          "self": "https://api.pagerduty.com/users/PMPOKBU",
          "html_url": "https://pdt-lucas.pagerduty.com/users/PMPOKBU"
        }
      ],
      "privilege": null,
      "schedule_layers": [
        {
          "name": "Layer 1",
          "rendered_schedule_entries": [],
          "rendered_coverage_percentage": null,
          "id": "PIBVIHP",
          "start": "2016-11-21T13:19:52-08:00",
          "end": null,
          "rotation_virtual_start": "2016-11-21T13:00:00-08:00",
          "rotation_turn_length_seconds": 604800,
          "users": [
            {
              "user": {
                "id": "PMPOKBU",
                "type": "user_reference",
                "summary": "Delete User Test User",
                "self": "https://api.pagerduty.com/users/PMPOKBU",
                "html_url": "https://pdt-lucas.pagerduty.com/users/PMPOKBU"
              }
            }
          ],
          "restrictions": []
        }
      ],
      "overrides_subschedule": {
        "name": "Overrides",
        "rendered_schedule_entries": [],
        "rendered_coverage_percentage": null
      },
      "final_schedule": {
        "name": "Final Schedule",
        "rendered_schedule_entries": [],
        "rendered_coverage_percentage": null
      },
      "teams": []
    }
  ],
  "get_escalation_policy": [
    {
      "id": "PUT4PSP",
      "type": "escalation_policy",
      "summary": "Delete User Test EP",
      "self": "https://api.pagerduty.com/escalation_policies/PUT4PSP",
      "html_url": "https://pdt-lucas.pagerduty.com/escalation_policies/PUT4PSP",
      "name": "Delete User Test EP",
      "escalation_rules": [
        {
          "id": "P2MLJ1X",
          "escalation_delay_in_minutes": 30,
          "targets": [
            {
              "id": "PMPOKBU",
              "type": "user_reference",
              "summary": "Delete User Test User",
              "self": "https://api.pagerduty.com/users/PMPOKBU",
              "html_url": "https://pdt-lucas.pagerduty.com/users/PMPOKBU"
            }
          ]
        }
      ],
      "services": [],
      "num_loops": 0,
      "teams": [
        {
          "id": "PU30AUS",
          "type": "team_reference",
          "summary": "Delete User Test Team",
          "self": "https://api.pagerduty.com/teams/PU30AUS",
          "html_url": "https://pdt-lucas.pagerduty.com/teams/PU30AUS"
        }
      ],
      "description": "",
      "privilege": null
    }
  ],
  "check_schedule_for_user": [
    true,
    false
  ],
  "check_team_for_user": [
    true,
    false
  ],
  "get_user_layer_index": [
    1,
    null
  ],
  "get_target_indices": [
    [
      {
        "rule": 0,
        "target": 1
      },
      {
        "rule": 2,
        "target": 0
      }
    ],
    [
      {
        "rule": 1,
        "target": 2
      }
    ]
  ],
  "remove_user_from_layer": [
    {
      "id": "XXXXXX",
      "name": "Layer 1",
      "users": [
        {
          "user": {
            "id": "AAAAAA",
            "type": "user",
            "name": "Arya Stark"
          }
        },
        {
          "user": {
            "id": "CCCCCC",
            "type": "user",
            "name": "Robb Stark"
          }
        }
      ]
    }
  ],
  "remove_from_escalation_policy": [
    [
      {
        "id": "AAAAAA",
        "targets": [
          {
            "id": "BBBBBB",
            "type": "user",
            "name": "Arya Stark"
          }
        ]
      },
      {
        "id": "CCCCCC",
        "targets": [
          {
            "id": "BBBBBB",
            "type": "user",
            "name": "Arya Stark"
          }
        ]
      },
      {
        "id": "DDDDDD",
        "targets": [
          {
            "id": "BBBBBB",
            "type": "user",
            "name": "Arya Stark"
          },
          {
            "id": "EEEEEE",
            "type": "user",
            "name": "Sansa Stark"
          },
          {
            "id": "FFFFFF",
            "type": "schedule",
            "name": "House Stark"
          },
          {
            "id": "GGGGGG",
            "type": "user",
            "name": "Robb Stark"
          }
        ]
      }
    ]
  ],
  "cache_schedule": [
    [
      {
        "id": "AAAAAA",
        "name": "House Lannister"
      },
      {
        "id": "BBBBBB",
        "name": "House Baratheon"
      },
      {
        "id": "ABCDEF",
        "name": "House Stark"
      }
    ]
  ],
  "cache_team": [
    [
      {
        "id": "AAAAAA",
        "name": "House Lannister"
      },
      {
        "id": "BBBBBB",
        "name": "House Baratheon"
      },
      {
        "id": "ABCDEF",
        "name": "House Stark"
      }
    ]
  ]
}
